sample-config.json

```diff
@@ -0,0 +1,10 @@
+{
+  "arweaveKeysFile": "./secrets/limestone-jwk.json",
+  "manifestFile": "./sample-manifests/stocks.json",
+  "credentials": {
+    "alphavantageApiKey": "XXX",
```

Comment:
Do we use alphavantage and finnhub as they're missing from the Credentials object interface?